Makefile

```
@@ -1,4 +1,113 @@
+	-X 'github.com/konveyor-ecosystem/kubectl-migrate/internal/buildinfo.Commit=$(COMMIT)' \
+	-X 'github.com/konveyor-ecosystem/kubectl-migrate/internal/buildinfo.BuildDate=$(BUILD_DATE)'"
+
+# Platforms for cross-compilation
+PLATFORMS = linux/amd64 linux/arm64 darwin/amd64 darwin/arm64
+
+.PHONY: all build build-all clean test deps install resources-deploy resources-destroy resources-validate help
+
+all: build
+
+## help: Display this help message
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (7).

(maxbodylength)